Alarmi/Alarmi/Sources/MainTab/Today/TodayViewController.swift

```diff
@@ -17,103 +17,133 @@ final class TodayViewController: UIViewController {
 
     // MARK: View
     
-    private let stackView: UIStackView = {
+    private let clockStackView: UIStackView = {
         $0.translatesAutoresizingMaskIntoConstraints = false
         $0.axis = .vertical
         $0.distribution = .fill
         $0.alignment = .leading
-        $0.spacing = 8
+        $0.spacing = -10
         return $0
     }(UIStackView())
 
     private let descriptionLabel: UILabel = {
         $0.translatesAutoresizingMaskIntoConstraints = false
-        $0.text = "마지막으로 전화한지 5일이 지났어요."
-        $0.setDynamicFont(.title2)
+        $0.text = "지금 한국은"
+        $0.setDynamicFont(for: .body, weight: .semibold)
         return $0
     }(UILabel())
 
-    private let subDescriptionLabel: UILabel = {
+    private let statusDescriptionLabel: UILabel = {
         $0.translatesAutoresizingMaskIntoConstraints = false
-        $0.text = "2일 후면 전화하기로 한 날이에요."
-        $0.textColor = .secondaryLabel
-        $0.setDynamicFont(.body)
+        $0.text = "전화 가능 시간이 아니에요."
+        $0.textAlignment = .right
+        $0.setDynamicFont(for: .body, weight: .bold)
         return $0
     }(UILabel())
 
-    private lazy var delayButton: UIButton = {
+    private let realTimeClockLabel: UILabel = {
         $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.adjustsFontSizeToFitWidth = true
+        $0.textAlignment = .center
+        $0.font = UIFont.systemFont(ofSize: 108, weight: .black)
+        return $0
+    }(UILabel())
```

Comment:
p1;

<img width="150" src="https://user-images.githubusercontent.com/75792767/181877031-79f1f906-70a2-49c8-9de7-068bb23a0984.gif">

realTimeClockLabel에 기본값이 할당되어있지 않아 앱이 실행되고 나서 1초가 지날때까지 라벨이 시계 없이 나타나게 됩니다. 기본값을 추가해주세요.
```suggestion
    private lazy var realTimeClockLabel: UILabel = {
        $0.translatesAutoresizingMaskIntoConstraints = false
        $0.adjustsFontSizeToFitWidth = true
        $0.textAlignment = .center
        $0.text = dateFormatter.string(from: Date())
        $0.font = UIFont.systemFont(ofSize: 108, weight: .black)
        return $0
    }(UILabel())
```

Reply:
헐 이게 이런 원인이었군요, 안그래도 이부분에 대해서 따로 올려야겠다고생각했는데, 감사합니다! 수정해서 커밋날릴게요

Reply:
감사합니다 수정완료했습니당 [4beeecf](https://github.com/DeveloperAcademy-POSTECH/MC3-Team7-MoTe/pull/97/commits/4beeecfd71c85104fa8bd37912a97c7c630c5d4f)